pkgs/applications/graphics/ImageMagick/7.0.nix

```diff
@@ -27,6 +27,8 @@ stdenv.mkDerivation rec {
     sha256 = "0r8zmk2cfmf09l94hqzfz4aspnzn178ggdbgm7w4hr0p864cbvc3";
   };
 
+  patches = [ ./revert-075565e93c71bcaaabf0ce70b7d1060bccdf0020.patch ];
```

Comment:
Was there an issue filed upstream about this? If so, could you please add a link to it in a comment above this patch?

Reply:
Yes, and upstream provided a fix within minutes. Will update the PR with the upstream patch.